opm/autodiff/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -195,7 +202,6 @@ class SimulatorFullyImplicitBlackoilEbos
             }
         }
         std::vector<std::vector<double>> OOIP;
-
```

Comment:
no, but I suppose that the empty line is not there there in the flow_legacy version ;)